social-proof-section-master/styles.css

```diff
@@ -0,0 +1,275 @@
+html {
+  height: 100%;
+}
+
+body {
+  background-color: hsl(0, 0%, 100%);
+  color: hsl(300, 43%, 22%);
+  font-size: 15px;
+  font-family: "League Spartan";
+  font-weight: 400;
+  height: 100%;
+  line-height: 1.2;
+}
+
+h1 {
+  font-weight: 700;
+  font-size: 3em;
+  line-height: 1;
+}
+
+p {
+  line-height: 1.5;
+  font-weight: 500;
+}
+
+#bg-top {
+  background: no-repeat url("./assets/images/bg-pattern-top-desktop.svg");
+  width: 100%;
+  height: 100%;
+  background-size: 35%;
+  background-position: top left;
+  position: fixed;
+  z-index: -1000;
+  top: 0;
+  left: 0;
+  right: 0;
+}
+
+#bg-bottom {
+  background: no-repeat url("./assets/images/bg-pattern-bottom-desktop.svg");
+  width: 100%;
+  height: 100%;
+  background-size: 65%;
+  background-position: bottom right;
+  position: fixed;
+  z-index: -1000;
+  bottom: 0;
+  left: 0;
+  right: 0;
+}
+
+#container {
```

Comment:
On ultra wide screens / high resolution the page looks kind of stretched out:
<img width="1068" alt="image" src="https://github.com/wojdanowski/exercises/assets/100123175/f727181b-f2ae-475e-963f-7ace75d05669">

I think it would be better to constrain content in a certain area as shown in `desktop-design.jpg` and with extra screen width just increase margins